pkg/operator/tlsrotationcontroller.go

```diff
@@ -0,0 +1,166 @@
+// Copyright 2018 The Cluster Monitoring Operator Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package operator
```

Comment:
So we are introducing a new controller, curious why can't this be part of the original pattern of tasks reconcile we already have?

Reply:
`TLSRotationController` has its own reconcilation but I agree it would be nice, potentially, to have a dedicated task which dispatches to the reconciliation callback of the external TLS controller such that we still have a single point of dispatch and control the assets that we are observing within CMO.